storybook/constants.js

```diff
@@ -6,6 +6,7 @@ export const BASE_URL = {
 export const DOCUMENTATION_URL = {
   PATTERNFLY_ORG_COMMUNICATION: `${BASE_URL.PATTERNFLY_ORG}communication/`,
   PATTERNFLY_ORG_WIDGETS: `${BASE_URL.PATTERNFLY_ORG}widgets/`,
+  PATTERNFLY_ORG_STYLES: `https://www.patternfly.org/styles/`,
```

Comment:
Shouldn't it be?
```jsx
PATTERNFLY_ORG_STYLES: `${BASE_URL.PATTERNFLY_ORG}styles/`,
```

Reply:
@sharvit I agree, but the 
`BASE_URL.PATTERNFLY_ORG`
is equal to 
'http://www.patternfly.org/pattern-library/'
and I didn't want to brake the code.

Should we add another constant called 'PATTERN_LIBRARY' ? 
and set the `PATTERNFLY_ORG` to be just 'http://www.patternfly.org' ?

Reply:
Would make sense to have

PATTERNFLY_ORG = 'http://www.patternfly.org'
PATTERNFLY_LIBRARY = PATTERNFLY_ORG/pattern-library/
PATTERNFLY_STYLES = PATTERNFLY_ORG/styles/

Though we would then need to update all current uses of PATTERNFLY_ORG  😢

Reply:
whoops!! apologies... well my editor has global find/replace, does yours? Feel free to do this...

Reply:
oh - so it looks like `PATTERNFLY_ORG` was not actually being used everywhere and this is OK now...phew!